src/adoptawidget/index.md

```diff
@@ -0,0 +1,280 @@
+///    ); 
+///  }
+/// ```
+/// {@end-tool}
+```
+
+### Adding References {#adding-references}
+
+If a Widget needs references, add a `See also:` section to the end of a widget's
+documentation comment:
+
+```dart
+/// Creates a route for the given route settings.
+///
+/// Used by [Navigator.onGenerateRoute].
+///
+/// See also:
+///
+///  * [Navigator], which is where all the [Route]s end up.
+```
+
+### Adding more explanation to the documentation {#adding-more-explanation-to-the-documentation}
+
+If a widget doesn't provide enough explanation, you can add more! You can
+explain:
+
+*   What it is
+*   What it enables you to do
+*   How to use it
+*   The behavior of the widget in certain situations
+*   Other important information about the widget
+
+See the [Documentation
+section](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc)
+of the Flutter Style Guide and [Effective Dart:
+Documentation](https://dart.dev/guides/language/effective-dart/documentation)
+for details on how to write great documentation.
+
+## Quickstart {#quickstart}
+
+### Get the code {#get-the-code}
+
+To get started, fork the SDK and use Git to fetch a local copy:
+
+1. Install dependencies listed in
+   [Setting up the Framework development environment](https://github.com/flutter/flutter/wiki/Setting-up-the-Framework-development-environment)
+2. Fork the flutter repo
+[https://github.com/flutter/flutter](https://github.com/flutter/flutter). For
+more information, see [GitHub's official
+instructions](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo).
+3. Clone your fork: `git clone
+https://github.com/&lt;your_name_here>/flutter.git`
+4. Change directory: `cd flutter`
+5. Add an upstream remote: `git remote add upstream
+git@github.com:flutter/flutter.git`
+6. Run `flutter update-packages` to fetch all the Dart packages that Flutter
+uses.
+
+
+### Open the code in your IDE {#open-the-code-in-your-ide}
+
+In IntelliJ or VSCode, open `packages/flutter`, where most of the Flutter
+framework code lives.
+
+
+### Make the change {#make-the-change}
+
+Open the file listed in the issue and update the documentation comment. See
+"Ways to contribute" for examples.
+
+### Analyzing and Generating API Docs
+
+Running this command from the root directory analyzes and generates the
+documentation. This command can take up to 20 minutes to complete, so double
+check your changes first. If you want, you can also [analyze
+snippets](https://github.com/flutter/flutter/tree/master/dev/snippets#running-sample-analysis-locally)
+directly.
+
+```bash
+./dev/bots/docs.sh
+```
+
+To view the generated docs, open `dev/docs/doc/api/index.html`.
+
+**Note:** Flutter API docs cannot be generated on Windows. If you are using
+*Windows, you may need to download the Linux version of the SDK and use the
+*[Windows Subsystem for
+*Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
```

Comment:
This is rendering weirdly. It appears that you are trying to create a code-style note, but it doesn't work as you think it does in markdown.

Instead:

{{site.alert.note}}
  Flutter API docs cannot be generated on Windows. If you are using
  Windows, you may need to download the Linux version of the SDK and use the
  [Windows Subsystem for
  Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
{{site.alert.end}}

You know, we don't actually do our links inline like this, but this is a temporary page, so I'm going to ignore that.  :)